src/Components/Artwork/Details.tsx

```diff
@@ -112,6 +112,10 @@ export class Details extends React.Component<Props, null> {
   }
 
   saleInfoLine() {
+    if (!this.props.showSaleLine) {
```

Comment:
Is this something that was missed somewhere else, or is this part of this work? I don't see `showSaleLine` anywhere else within the GitHub UI, but that doesn't mean it's not hiding it from me...

Reply:
This piggy backs on existing props in this UI that isn't actually being used anywhere at the moment (but did in the past). It refers to the sale state -- so `Bidding Open`, `Sold`, etc. We don't want to include that line in the artwork brick for consignments.